sdk/communication/azure-communication-sms/pom.xml

```diff
@@ -192,5 +192,31 @@
         <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
       </properties>
     </profile>
+    <profile>
+      <id>sms-coverage</id>
```

Comment:
Any reason for creating a custom profile for getting code coverage report instead of using the central code coverage setup?

Reply:
We are generating coverage report for a single package/sdk. I'm not sure how this can be done. Just experimenting. Can you please add a bit more details on "the central code coverage setup"? I looked at "eng\jacoco-test-coverage" briefly. No idea how that can be adapted for our purpose.